Src/PRuntimes/PExplicitRuntime/src/main/java/pexplicit/values/ComputeHash.java

```diff
@@ -43,4 +45,18 @@ public static int getHashCode(PMachine machine, PValue<?>... values) {
         }
         return hashValue;
     }
+
+    /**
+     * Compute hash value for a PMachine local variables.
+     */
+    public static int getHashCode(SortedSet<PMachine> machines) {
```

Comment:
I am hoping this way of computing hash is prototype and we improve on this?

Reply:
Yes, indeed. A much improved version of state caching to follow in the next PR.